nixos/modules/services/misc/nixos-manual.nix

```diff
@@ -58,6 +58,15 @@ let
       exec "$browser" ${entry}
     '';
 
+  desktopItem = pkgs.makeDesktopItem {
+    name = "nixos-manual";
+    desktopName = "NixOS Manual";
+    genericName = "View NixOS documentation in a web browser";
+    # TODO: find a better icon (Nix logo + help overlay?)
+    icon = "system-help";
+    exec = "xdg-open ${entry}";
```

Comment:
Desktop entries [specifically allow URLs](https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys), so I guess we might just use `file://${entry}` URL instead of `xdg-open`.